mmv1/templates/terraform/custom_expand/containerattached_cluster_fleet_project.go.erb

```diff
@@ -0,0 +1,22 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2022 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func expand<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
```

Comment:
This expander doesn't seem to match the behavior of the AWS & Azure clusters. Does the API reject requests that are not prepended with `projects/`?

Reply:
That is correct. I personally tend to forget about that requirement so I figured it'd be nice to have this additional help.

Reply:
this does seem convenient, but I'm concerned about then getting requests to add it on the AWS and Azure versions (which would be trickier.) alternatively, you could add a validation function like [StringMatch](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/helper/validation#StringMatch) that ensures that the provided value starts with `projects/`... what do you think?

Reply:
I added a regex validation to check -- it seemed simplest.